### ISSUE TITLE
Is only Kinetics-400 used for training in the Kinetics dataset?

### ISSUE THREAD

**Issue by Parsifal133** [farewellthree/PPLLaVA#12]
Hi, @farewellthree ！

I am trying to reproduce the training results recently, and I notice that [the only Kinetics json you provided](https://huggingface.co/datasets/farewellthree/ppllava_json/tree/main) was `train_k400.json`, which seems to indicate that only **Kinetics-400** was used for training, and not **Kinetics-600, Kinetics-700**, etc. 

I cannot verify this detail in the paper or readme. Please correct me if I am wrong.

**Comment by Parsifal133** [comment]
And I tried to reproduce it with 3 a800 graphics cards. No matter how I adjusted the batchsize and gradient accumulation, the training would get stuck until it timed out after the first batch of data was forward propagated. This problem disappeared until I only used video training.

**Comment by farewellthree** [comment]
Yes, we only used Kinetics-400, and theoretically, the results would be better with Kinetics-910. However, due to the more complicated data collection process for Kinetics-910, we simplified it.

**Comment by Parsifal133** [comment]
Thanks for the wonderful reply, it solved my problem.
